### PR TITLE
Add codeowners for automate reviewer assignment

### PR DESCRIPTION
**Pull request by turbaszek**
> Code owners are automatically requested for review when someone opens a pull request that modifies code that they own. 
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

I think we may try this feature. Currently contributors are not able to request review (unless they ping a committer). 

I tried to create some basic rules but I'm happy to get any suggestions!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# All repo
+* @turbaszek
```
Comment (turbaszek) [inline .github/CODEOWNERS]: I did not added all committers because not everyone may want to get updates. Let me know if I should add you

**Review by sharanf — APPROVED** [review]
Good idea to try it out and I think it will be useful.

File: .github/CODEOWNERS (hunk 1 above)
Comment (sharanf) [reply .github/CODEOWNERS]: I'm happy to manage the items you put me down for :-)

Comment (michalslowikowski00) [reply .github/CODEOWNERS]: I will do my best to support. Please add me as well, Tomek.

**Review by michalslowikowski00 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (turbaszek) [inline .github/CODEOWNERS]:
```suggestion
* @turbaszek @michalslowikowski00
```

**Pull request merged**